.github/workflows/check-manifests.yml

```diff
@@ -26,12 +26,12 @@ jobs:
     - name: Check x86_64 manifests
       run: |
         pushd toolkit
-        make check-x86_64-manifests
+        ./scripts/toolchain/check_manifests.sh -a "x86_64"
         popd
 
     - name: Check aarch64 manifests
       run: |
         pushd toolkit
-        make check-aarch64-manifests
+        ./scripts/toolchain/check_manifests.sh -a "aarch64"
         popd
       if: always()
```

Comment:
What I have been trying to do is add a corrective action for the user into the pipeline.
Something like:
```bash
echo ##########
echo "Ensure toolchain and pkggen manifests (./toolkit/resources/manifests/package/*) match the versions in the .spec files"
echo "Run 'make check-manifests' to validate locally"
echo ##########
```